proto/ignition/msgs/bounding_box.proto

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "BoundingBoxProtos";
+
+/// \ingroup ignition.msgs
+/// \interface BoundingBox
+/// \brief BoundingBox message for object detection annotation
+
+import "ignition/msgs/header.proto";
+
+message BoundingBox
+{
+  /// \brief Optional header data
+  Header header = 1;
+
+  /// \brief Min x coord of the box (top left corner)
+  uint32 minX = 2;
+
+  /// \brief Min y coord of the box (top left corner)
+  uint32 minY = 3;
+
+  /// \brief Max x coord of the box (bottom right corner)
+  uint32 maxX = 4;
+
+  /// \brief Max y coord of the box (bottom right corner)
+  uint32 maxY = 5;
```

Comment:
Can we also add `min` / `max` `Z` values so the message can be used for 3D boxes too?

Reply:
I think we should actually have separate message types for 2D and 3D messages - see https://github.com/ignitionrobotics/ign-sensors/issues/135#issuecomment-872480774